### PR TITLE
OSDOCS-6778

### PR DESCRIPTION
**Pull request by smunje1**
[OSDOCS-6778](https://issues.redhat.com/browse/OSDOCS-6778): Service Mesh Operators - Clarity

Aligned team: Service Delivery
OCP version for cherry-picking: enterprise-4.14+
JIRA issues: [OSDOCS-6778](https://issues.redhat.com/browse/OSDOCS-6778)
Preview pages: [ROSA build preview](https://70904--ocpdocs-pr.netlify.app/openshift-rosa/latest/service_mesh/v2x/installing-ossm#ossm-installation-activities_installing-ossm)
SME review **completed**: @arendej
QE review **completed**:  @gwynnemonahan
Peer review **completed**: @nalhadef

### CODE REVIEW

**Review by arendej — APPROVED** [review]
lgtm

File: modules/ossm-installation-activities.adoc [hunk 1]
```diff
@@ -8,9 +8,9 @@
...
 * *{JaegerName}* - Provides distributed tracing to monitor and troubleshoot transactions in complex distributed systems. It is based on the open source link:https://www.jaegertracing.io/[Jaeger] project.
 * *Kiali Operator provided by Red Hat* - Provides observability for your service mesh. You can view configurations, monitor traffic, and analyze traces in a single console. It is based on the open source link:https://www.kiali.io/[Kiali] project.
 * *{SMProductName}* - Allows you to connect, secure, control, and observe the microservices that comprise your applications. The {SMProductShortName} Operator defines and monitors the `ServiceMeshControlPlane` resources that manage the deployment, updating, and deletion of the {SMProductShortName} components. It is based on the open source link:https://istio.io/[Istio] project.
+* *OpenShift Elasticsearch* - (Optional) Provides database storage for tracing and logging with the {JaegerShortName}. It is based on the open source link:https://www.elastic.co/[Elasticsearch] project.
```
Comment (gwynnemonahan) [inline modules/ossm-installation-activities.adoc:16]: Though OpenShift Elasticsearch is optional, if it is going to be used, it must be installed first. In order to be consistent throughout Service Mesh, OpenShift Elasticsearch must remain first in the list.

Comment (smunje1) [reply modules/ossm-installation-activities.adoc]: Revoked this change.

**Comment by smunje1** [comment]
Hello @gwynnemonahan . Thanks for your input. I have incorporated your suggestion.
Could you please take a look now? Thanks!

File: modules/ossm-installation-activities.adoc [hunk 2]
```diff
@@ -8,7 +8,7 @@
 [id="ossm-installation-activities_{context}"]
 = Operator overview
 
-{SMProductName} requires the following four Operators:
+{SMProductName} requires the following Operators:
```
Comment (nalhadef) [inline modules/ossm-installation-activities.adoc:11]: I'd delete "following." It doesn't really add anything.

**Comment by nalhadef** [comment]
Looks good, @smunje1. Just one suggestion.

File: modules/ossm-installation-activities.adoc (hunk 2 above)
Comment (smunje1) [reply modules/ossm-installation-activities.adoc:11]: I believe we should keep "following". Just below this section we have, [Installing the Operators](https://docs.openshift.com/rosa/service_mesh/v2x/installing-ossm.html#ossm-install-ossm-operator_installing-ossm) section there also we can see the sentence, "To install Red Hat OpenShift Service Mesh, install the _following_ Operators in this order."

Please let me know your thoughts. Thanks!

Comment (smunje1) [reply modules/ossm-installation-activities.adoc:11]: Hello @nalhadef . I believe we should keep "following". Just below this section we have, [Installing the Operators](https://docs.openshift.com/rosa/service_mesh/v2x/installing-ossm.html#ossm-install-ossm-operator_installing-ossm) section there also we can see the sentence, "To install Red Hat OpenShift Service Mesh, install the _following_ Operators in this order."

Please let me know your thoughts. Thanks!

**Pull request merged**